components/charts/line-charts/DeveloperHoursLineChart.tsx

```diff
@@ -126,19 +48,68 @@ const legends = [
 ]
 const pointSize = 6
 const useMesh = true
+const yScale = {
+    type: 'linear',
+    min: 0,
+    max: 8
+}
+const colors = ['#3066be', '#da5b38', '#3fc1aa']
 
 const DeveloperHoursLineChart = () => {
+    const { devHours } = useSelector((state: any) => state.developerHours)
+    const { developers } = devHours
```

Comment:
Lines 59-60 can be consolidated into one.

```
const { devHours: { developers } } = useSelector((state: any) => state.developerHours)
```

OR

```
const {developers} = useSelector((state:any) => state.developerHours.developers)
```

Reply:
Not 100% sure the second one will work, so try it out just to see if it does. The first one is more tried and true so it can be the fallback